packages/flutter_tools/lib/src/commands/packages.dart

```diff
@@ -213,6 +213,7 @@ class PackagesGetCommand extends FlutterCommand {
     argParser.addOption('git-ref');
     argParser.addOption('git-path');
     argParser.addFlag('dev');
+    argParser.addFlag('verbose', abbr: 'v');
```

Comment:
Generally in the tool, we handle the `verbose` flag separately, before the arg parser: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/executable.dart#L64

Do we need to handle this here?

Reply:
It is not really being handled - just accepted by this preliminary parser.
Otherwise we would end up not understanding eg. '--dry-run -v'.